src/_guides/language/sound-problems.md

```diff
@@ -55,9 +54,9 @@ see [Runtime errors](#common-errors-and-warnings).
 
 ### Undefined member
 
-<?code-excerpt "strong/analyzer-results-stable.txt" retain="/isn't defined for the type.*common_problems/" replace="/getter/<member\x3E/g; /'\w+'/'...'/g"?>
+<!-- code-excerpt "strong/analyzer-results-stable.txt" retain="/isn't defined for the type.*common_problems/" replace="/getter/<member\x3E/g; /'\w+'/'...'/g" -->
 ```nocode
-error • The <member> '...' isn't defined for the type '...' • undefined_<member>
+-error • The <member> '...' isn't defined for the type '...' • undefined_<member>
```

Comment:
The `-` before `error` looks like a mistake... probably mine.

Reply:
Fixed in a850e36